### PR TITLE
WPC-1068: Embed block + other embeds blocks

### PR DESCRIPTION
**Pull request by antkiewiczk**

### CODE REVIEW

File: libs/block-content/website/src/lib/tik-tok-video-block.tsx [hunk 1]
```diff
@@ -7,6 +7,27 @@ export const isTikTokVideoBlock = (block: Block): block is TikTokVideoBlockType
 
 export const TikTokVideoBlockWrapper = styled('div')``
 
+export const TikTokVideoBlockPlayer = styled('div')`
```
Comment (Itrulia) [inline libs/block-content/website/src/lib/tik-tok-video-block.tsx:10]: You can use react-player for this. See youtube or vimeo

**Review by Itrulia — COMMENTED** [review]
Is this in draft or ready for review?

**Comment by antkiewiczk** [comment]
> Is this in draft or ready for review?

Marked it as ready, but the tests are failing because react-player is changing the id every time when writing snapshot. I am now looking at solution for this. If you have anything else, let me know

File: libs/block-content/website/src/lib/tik-tok-video-block.tsx (hunk 1 above)
Comment (antkiewiczk) [reply libs/block-content/website/src/lib/tik-tok-video-block.tsx:10]: I used iframe directly, as react-player wouldn't run (some cross-origin issues)

**Comment by Itrulia** [comment]
@antkiewiczk many embeds are missing, hence im asking.

File: libs/block-content/website/src/lib/tik-tok-video-block.tsx (hunk 1 above)
Comment (Itrulia) [reply libs/block-content/website/src/lib/tik-tok-video-block.tsx:10]: My bad, didn't see Tiktok was not supported by react-player as it TikTok doesn't have a non iframe API.

Can we atleast use a CSS specifity of 1, not hardcode widths/heights and give it a proper title?

File: libs/block-content/website/src/lib/facebook-video-block.tsx [hunk 2]
```diff
@@ -24,6 +24,7 @@ export const FacebookVideoBlock = ({
       height={'auto'}
       url={`https://www.facebook.com/${userID}/videos/${videoID}/`}
       controls={true}
+      config={{facebook: {playerId: 'facebook-player'}}}
```
Comment (Itrulia) [inline libs/block-content/website/src/lib/facebook-video-block.tsx:27]: this now causes that you can not have the multiple facebook videos as they have the same Id. 
Please combine this with `const id = useId()`

File: libs/block-content/website/src/lib/bildwurf-ad-block.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    load()
+    try {
+      window._ASO.loadAd('bildwurf-injection-wrapper', zoneID)
+    } catch (error) {
+      console.warn('could not call _ASO.loadAd()')
+    }
+  }, [])
+
+  return (
+    <BildwurfBlockWrapper className={className} id="bildwurf-injection-wrapper">
```
Comment (antkiewiczk) [inline libs/block-content/website/src/lib/bildwurf-ad-block.tsx:88]: I tried using `const id = useId()`, but it wouldn't load

File: libs/block-content/website/src/lib/bildwurf-ad-block.tsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+import styled from '@emotion/styled'
+import {BuilderBildwurfAdBlockProps} from '@wepublish/website/builder'
+
+function useScript(src: string, checkIfLoaded: () => boolean, crossOrigin = false) {
+  const scriptRef = useRef<HTMLScriptElement | null>(null)
+
+  const [isLoading, setLoading] = useState(false)
+  const [isLoaded, setLoaded] = useState(() => checkIfLoaded())
+
+  useEffect(() => {
```
Comment (Itrulia) [inline libs/block-content/website/src/lib/bildwurf-ad-block.tsx:13]: Can we use `<Script>` instead? (It's in the website builder, probably won't work on storybook but should in the next app as the storybook mock is very basic)

File: libs/block-content/website/src/lib/bildwurf-ad-block.tsx (hunk 3 above)
Comment (Itrulia) [reply libs/block-content/website/src/lib/bildwurf-ad-block.tsx:88]: Just checked, while the script doesn't include it and even gives a way to specify it, another script that is loaded hardcodes it.

**Review by Itrulia — APPROVED** [review]

**Pull request merged**